### PR TITLE
Feature/vars

### PR DESCRIPTION
**Pull request by larnedo**
Updated fonts vars and font-size mixin.

### CODE REVIEW

File: src/_mixins.scss [hunk 1]
```diff
@@ -22,19 +22,20 @@
...
-  font-size: ($fz-default / $fz-base) * 1rem;
 
-  @if $lh-default == auto {
-     line-height: ceil($fz-default / $lh-base) * ($lh-base / $fz-default);
+  @if $lh-default == normal {
+    @if $fz-default > $fz-base {
+       line-height: ceil($fz-base * $lh-base-headings) - ($fz-base - $fz-default);
+    }
+
+    @elseif $fz-default <= $fz-base {
```
Comment (nucliweb) [inline src/_mixins.scss]: Change this line for:
```
@else {
```

Thanks

File: src/_vars.scss [hunk 2]
```diff
@@ -27,36 +27,32 @@ $c-dark-light:              #f2f2f2 !default;
 
 // Font families
 // -----------------------------------------------------------------------------
-$ff-sans-serif:             "Open Sans", Helvetica, Arial, sans-serif !default;
-$ff-base: 			        Proximanova, Helvetica, Arial, sans-serif !default;
-$ff-base-bold: 			    Proximanovabold, Helvetica, Arial, sans-serif !default;
+$ff-base: 			            "latoregular", Helvetica, Arial, sans-serif !default;
+$ff-base-bold: 			        "latobold", Helvetica, Arial, sans-serif !default;
```
Comment (nucliweb) [inline src/_vars.scss]: Please, align tabs.

Comment (larnedo) [inline src/_vars.scss]: I don't know why you can't see aligned, but they are :(

File: src/_mixins.scss (hunk 1 above)
Comment (larnedo) [inline src/_mixins.scss]: Fixed!

**Comment by nucliweb** [comment]
:+1:

**Pull request merged**